### PR TITLE
Retrieve changelogs from the Unitystation API

### LINKED ISSUE #178: Changelog view should instead show our own curated changelog
Currently, the changelog view displays all merged PRs, which a lot of the time isn't really relevant for players. It should instead list the changes we have curated and manually recorded in our changelog api, currently living on https://changelog.unitystation.org/whats-new (for only latest build) or https://changelog.unitystation.org/all-changes

### PR DESCRIPTION
**Pull request by CorruptComputer**
Retrieve changelogs from the Unitystation API instead of directly from GitHub.

image

Fixes #178

### CODE REVIEW

**Comment by CorruptComputer** [comment]
The last two Codacy issues are both false positives.

File: UnitystationLauncher/ViewModels/ChangeViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{
+    private string Type { get; }
+    private string Description { get; }
+    private string AuthorAndPr { get; }
+
+    public ChangeViewModel(Change change)
+    {
+        Type = $"[{change.Category}]";
+        Description = change.Description ?? "null";
+        AuthorAndPr = $"{change.AuthorUsername ?? "null"} | PR #{change.PrNumber}";
```
Comment (corp-0) [inline UnitystationLauncher/ViewModels/ChangeViewModel.cs:17]: can the PR be clickable and link to the PR on github?

Comment (CorruptComputer) [reply UnitystationLauncher/ViewModels/ChangeViewModel.cs:17]: Yeah, should be added now.

**Comment by corp-0** [comment]
last thing I would like to ask, if you don't mind. Could a build with no registered changes display a mesage like "No registered changes in this build"?

**Comment by CorruptComputer** [comment]
Yeah, should have that now as well:
image

**Review by corp-0 — APPROVED** [review]

**Pull request merged**